judo-runtime-core-export/src/main/java/hu/blackbelt/judo/runtime/core/export/JxlExportUtil.java

```diff
@@ -0,0 +1,205 @@
+package hu.blackbelt.judo.runtime.core.export;
+
+import hu.blackbelt.judo.dao.api.Payload;
+import hu.blackbelt.judo.meta.asm.runtime.AsmModel;
+import hu.blackbelt.judo.meta.asm.runtime.AsmUtils;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.poi.ss.usermodel.CellStyle;
+import org.apache.poi.ss.usermodel.CreationHelper;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.eclipse.emf.ecore.EClass;
+import org.jxls.area.XlsArea;
+import org.jxls.command.EachCommand;
+import org.jxls.common.CellRef;
+import org.jxls.common.Context;
+import org.jxls.transform.poi.SelectSheetsForStreamingPoiTransformer;
+
+import java.io.*;
+import java.lang.reflect.Method;
+import java.lang.reflect.ParameterizedType;
+import java.lang.reflect.Type;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.*;
+import java.util.stream.Collectors;
+
+@Slf4j
+public class JxlExportUtil {
+    private static String METHOD_PREFIX = "get";
+
+    public static Map<String, Class<?>> getAttributesFromClass(Class clazz) {
```

Comment:
This method and `getAttributesFromModel` should be at least `protected` because we are only using them to set up parameters for `createExcelExport`.

Reply:
We can use this another places.